build/server/edgemesh/05-configmap.yaml

```diff
@@ -8,7 +8,12 @@ metadata:
     kubeedge: edgemesh-server
 data:
   edgemesh-server.yaml: |
+    kubeAPIConfig:
+      burst: 200
+      contentType: application/vnd.kubernetes.protobuf
+      kubeConfig: "/root/.kube/config"
```

Comment:
edgemesh-server use rbac+serviceaccount, why do we need to mount kubeconfig?
@ZBoIsHere  What is your suggestion?

Reply:
yesterday, edgemesh-server failed to run and the log printing "/root/.kube/config" kubeconfig not exist, if edgemesh-server use rbac+serviceaccount, does it means edgemesh-server do not need to get file /root/.kube/config?

Reply:
As @ZBoIsHere said, without giving the kubeAPIConfig information, the server and agent will just crash at boot because they can't find the kubeconfig file. Adding that section to the YAML file fixed the problem. I suppose they have no permission to access the local filesystem?

Reply:
If we use rbac+serviceaccount, there is no need to mount kubeconfig. Kubelet will automatically mount the token into the pod, and then be able to establish a connection with the kube-apiserver.
```
[root@k8s-master ~]# kubectl get secret
NAME                  TYPE                                  DATA   AGE
default-token-btm86   kubernetes.io/service-account-token   3      5d18h
```

Reply:
Mmh, I understand. I can see using rbac+serviceaccount being the correct solution, but it doesn't work in my case. Without mounting the kubeconfig I can't run the server and the agent. They just crash with the error ` kubeconfig not exist`.
```
v00609018@fraporion00001:~$ kubectl get secret
NAME                                           TYPE                                  DATA   AGE
default-token-kgdfb                            kubernetes.io/service-account-token   3      56d
```

Reply:
hello, vcozzolino and Poorunga,  if the user use rbac+serviceaccount, he doesn't need to use kubeconfig and doesn't need to mount kubeconfig, if the user use kubeconfig to connect to api server, he need to mount kubeconfig in the edgemesh-server or edgemesh-agent yaml. 
In this [PR](https://github.com/kubeedge/edgemesh/pull/58), it solve the problem of this [issue](https://github.com/kubeedge/edgemesh/issues/47), that's why edgemesh-agent can not work well before
when this  [PR](https://github.com/kubeedge/edgemesh/pull/58) merged into main branch, you can deploy edgemesh-server and edgemesh-agent without define kubeconfig in the configmap